modules/sdk-api/src/bitgoAPI.ts

```diff
@@ -748,10 +750,44 @@ export class BitGoAPI implements BitGoBase {
     this._token = accessToken;
   }
 
+  public createAccountKeychain = async (password: string): Promise<any> => {
+    const keyData = this.keychains().create();
+    const hdNode = bitcoin.HDNode.fromBase58(keyData.xprv);
+
+    return await this.keychains().add({
+      source: 'ecdh',
+      xpub: hdNode.neutered().toBase58(),
+      encryptedXprv: this.encrypt({
+        password: password,
+        input: hdNode.toBase58(),
+      }),
+    });
+  };
+
+  private updateUserSettings(params: any): Promise<any> {
+    return this.put(this.url('/user/settings', 2)).send(params).result();
+  }
+
+  private ensureUserEcdhKeychainIsCreated = async (password: string): Promise<any> => {
+    const userSettings = await this.get(this.url('/user/settings')).result();
+    if (!userSettings.settings.ecdhKeychain) {
+      const newKeychain = await this.createAccountKeychain(password);
+      await this.updateUserSettings({
+        settings: {
+          ecdhKeychain: newKeychain.xpub,
+        },
+      });
+      userSettings.settings.ecdhKeychain = newKeychain.xpub;
+      return userSettings.settings;
+    }
+
+    return userSettings.settings;
+  };
```

Comment:
Whats the reasoning for this format vs  
```
private async ensureUserEcdhKeychainIsCreated(password: string): Promise<any>  {
```

I'd prefer if they where all the same format if there is no benefit of one vs the other.